### PR TITLE
Update pools.json

### PR DESCRIPTION
**Pull request by MrSol0**
add grlc.co

### CODE REVIEW

**Review by meric426 — CHANGES_REQUESTED** [review]
This pool is shit. Make it better before merging this PR.

**Comment by MrSol0** [comment]
shit? what are you talking about?

File: pools.json [hunk 1]
```diff
@@ -36,6 +36,12 @@
 		"searchStrings": [
 			"/Garlicsoup.xyz/"
 		]
+	}, {
+		"poolName": "grlc.co",
+		"url": "http://grlc.co/",
```
Comment (ryan-shaw) [inline pools.json:41]: Get some HTTPS on this then I'll merge.

**Comment by meric426** [comment]
Also it's run entirely by ASIC's. It won't be around after the fork, no need to merge this.

**Pull request closed without merging**